### DISCUSSION TITLE
Following Not Updating

### DISCUSSION THREAD

**Discussion by sweetpearacer** [mastodon/mastodon#21001] [Q&A]
New user. Trying to follow accounts but my “following” number and home tab are not updating on app or web. Any thoughts on a fix? Thanks!

**Comment by ebresie** [comment]
> New user. Trying to follow accounts but my “following” number and home tab are not updating on app or web. Any thoughts on a fix? Thanks!

I also noticed when trying to view other people's profile that clicking the following doesn't seem to work for me either.
